### ISSUE TITLE
disable no-shadowed-variable with unused parameter (_)

### ISSUE THREAD

**Issue by PMExtra** [palantir/tslint#4908]
### Feature request

A code example:
```Javascript

// config.get = (key, (err, value) => void);

config.get("someFeature.available", (_, available) => {
  if (!available) return;
  config.get("someFeature.enabled", (_, enabled) => {
    if (!enabled) return;
    config.get("someFeature.url" (_, url) => {
      service.call(url, ...);
    });
  });
});
```

In this case, `config.get` is provided by 3rd party, and I want just ignored the error. So I used `_` as a placeholder for `err` parameter.
But TSLint warn me that `Shadowed name '_'`.
I think it would be perfect if we can except the `_` variable for this rule.

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is deprecated** and no longer accepting pull requests other than security fixes. See #4534. ☠️ 
We recommend you instead use **[typescript-eslint](https://typescript-eslint.io)** to lint your TypeScript code with **ESLint**. ✅

👋 It was a pleasure open sourcing with you!

**Comment by PMExtra** [comment]
@JoshuaKGoldberg Thank you, it's a Angular project, it's not support ESLint now.
I will continue to focus on the migration progress.
Ref:
https://github.com/angular/angular-cli/issues/13732
https://github.com/mgechev/codelyzer/issues/763

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
